server/app/assets/javascripts/enumerator.ts

```diff
@@ -0,0 +1,174 @@
+// Javascript handling for enumerators
+
+window.addEventListener('load', () => {
+  // Configure the button on the enumerator question form to add more enumerator field options
+  const enumeratorOptionButton = document.getElementById(
+    'enumerator-field-add-button',
+  )
+  if (enumeratorOptionButton) {
+    enumeratorOptionButton.addEventListener('click', addNewEnumeratorField)
+  }
+
+  // Configure existing enumerator entity remove buttons
+  addEventListenerToElements(
```

Comment:
This function is defined in main.ts as global function and used in this file. That's not great as function should be properly imported. But currently our TS is not setup in a way that supports it. So I guess that's fine. But can you add a comment here explaining that? Because it's pretty unusual and engineers might be confused on why this function works.

Reply:
Ah I just blindly copied and pasted things into this file. I didn't even realize this was coming from main.js. Thanks I added a comment!